R/src_sqlite.R

```diff
@@ -0,0 +1,43 @@
+#' Setup a sqlite database connection
+#'
+#' @export
+#' 
+#' @param dbname (character) name of database file
+#' @param ... additional named params passed on to [RSQLite::SQLite()]
+#' 
+#' @details uses \pkg{RSQLite} under the hood; uses [RSQLite::SQLite()] for
+#' connecting
+#' 
+#' @examples \dontrun{
+#' (con <- src_sqlite())
+#' print(con)
+#' }
+src_sqlite <- function(dbname = "test.sqlite", 
```

Comment:
at least in examples and tests, we can't have the sqlite file written to disk as is. for examples and tests, write to a temp directory.